docs/windows-design.md

```diff
@@ -24,12 +24,13 @@ SNAT based on OpenFlow is needed to make sure the containers can access the exte
 The SNATed address is using the IP configured on the OVS bridge. Some additional OpenFlow entries
 are installed to assist in identifying and forwarding the external traffic.
 
-kube-proxy is running on each Windows Node to implement Kubernetes Service. Service traffic is
-output to the host on gw0, and captured by kube-proxy. kube-proxy sets up a new connection to
-the backend endpoint, and forwards the request using this new connection. The forwarded request
-enters OVS pipeline again and is forwarded to the actual Pod. To be compatible with OVS, kube-proxy
-on Windows must be configured to run in **userspace** mode, and a specific network adapter is 
-required, on which Service IP addresses will be configured by kube-proxy.
+Antrea implements the Kubernetes ClusterIP Service leveraging OVS. And kube-proxy is running
```

Comment:
Add a short description about how to control the traffic from Pod to a ClusterIP Service, for such traffic is forwarded inside the OVS pipeline which is much different from the one controlled by kube-proxy.

Reply:
Done.